docs/office.md

```diff
@@ -121,6 +121,66 @@ Content-Type: application/vnd.api+json
 }
 ```
 
+### GET /office/keys/:key
```

Comment:
Shouldn't this endpoint rather be named something like `/office/files/:key` since it returns an `io.cozy.file` and not a key?

Reply:
I would expect a `POST` verb here since this route can create a new file.

Reply:
I would prefer to keep `/office/keys/:key`, but you are right that it should be a `POST`

Reply:
@cballevre I have changed the HTTP verb to `POST`